app/src/main/java/xyz/whereuat/whereuat/MainActivity.java

```diff
@@ -87,7 +94,14 @@ protected void onCreate(Bundle savedInstanceState) {
         this.startService(intent);
     }
 
-    //
+    /**
+     * This is called when the loader is first made. It is necessary to implements the LoaderManager
+     * callbacks.
+     *
+     * @param id the id of the loader, which is not needed because there is only one
+     * @param args miscellaneous arguments to help make the loader, of which there are none
```

Comment:
Not totally sure what they're for lol. Just an overridden function that needs them